static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -420,6 +421,14 @@ class GroupHeader extends Component<Props, State> {
                 {t('Similar Issues')}
               </ListLink>
             )}
+            <Feature features={['session-replay']} organization={organization}>
+              <ListLink
+                to={`${baseUrl}replays/${location.search}`}
+                isActive={() => currentTab === Tab.REPLAYS}
+              >
+                {t('Replays')} <Badge text={group.userReportCount} />
```

Comment:
This badge isn't correct

Reply:
this badge is not the final one, saw we are retrieving this userReportCount from a python file, don't know if should we do the same here. @billyvg